lib/workingDirectory.js

```diff
@@ -194,9 +194,6 @@ async function cloneRepo(
     "git clone -b " +
     branch +
     " --single-branch " +
-    " --depth " +
-    depth +
```

Comment:
👍 This change makes sense to keep things simple.

Reply:
Thanks! Version number change reverted.